packages/agoric-cli/src/commands/inter.js

```diff
@@ -83,56 +141,144 @@ export const fmtBid = (bid, assets) => {
 
   const {
     id,
-    error,
     proposal: { give },
     offerArgs: { want },
     payouts,
+    result,
+    error,
   } = bid;
+  const resultProp =
+    !error && result && result !== 'UNPUBLISHED' ? { result } : {};
   const props = {
     ...(give ? { give: fmt.record(give) } : {}),
     ...(want ? { want: fmt.amount(want) } : {}),
-    ...(payouts ? { payouts: fmt.record(payouts) } : {}),
+    ...(payouts ? { payouts: fmt.record(payouts) } : resultProp),
     ...(error ? { error } : {}),
   };
   return harden({ id, ...spec, ...props });
 };
 
 /**
+ * Make Inter Protocol liquidation bidding commands.
+ *
  * @param {{
  *   env: Partial<Record<string, string>>,
  *   stdout: Pick<import('stream').Writable,'write'>,
  *   stderr: Pick<import('stream').Writable,'write'>,
  *   now: () => number,
  *   createCommand: // Note: includes access to process.stdout, .stderr, .exit
  *     typeof import('commander').createCommand,
- *   execFileSync: typeof import('child_process').execFileSync
+ *   execFileSync: typeof import('child_process').execFileSync,
+ *   setTimeout: typeof setTimeout,
  * }} process
  * @param {{ fetch: typeof window.fetch }} net
  */
 export const makeInterCommand = async (
-  { env, stdout, stderr, now, execFileSync, createCommand },
+  {
+    env,
+    stdout,
+    stderr,
+    now,
+    setTimeout,
+    execFileSync: rawExec,
+    createCommand,
+  },
   { fetch },
 ) => {
   const interCmd = createCommand('inter')
-    .description('Inter Protocol tool')
-    .option('--home [dir]', 'agd CosmosSDK application home directory')
+    .description('Inter Protocol commands for liquidation bidding etc.')
+    .option('--home <dir>', 'agd CosmosSDK application home directory')
     .option(
       '--keyring-backend [os|file|test]',
-      'keyring\'s backend (os|file|test) (default "os")',
+      `keyring's backend (os|file|test) (default "${
+        env.AGORIC_KEYRING_BACKEND || 'os'
+      }")`,
+      env.AGORIC_KEYRING_BACKEND,
+    );
+
+  /** @type {typeof import('child_process').execFileSync} */
+  // @ts-expect-error execFileSync is overloaded
+  const execFileSync = (file, args, ...opts) => {
```

Comment:
awkward to rename this canonical function. instead please leave `rawExec` as `execFileSync` and name this like `friendlyExecFile`.

though is there anything for it to exec besides agd? better then to attenuate the power with `execAgd` that doesn't take `file`.

Reply:
on second thought, I don't think it's worth the bother. I'm adding a sketch of the POLA-shaped I/O design in a comment.

Reply:
ok, that design lgtm and I'll accept that as future work. 

I'd still like to be clear that we don't use canonical names like `execFileSync` for functions that have slightly different behavior. So consider an XXX here to that effect. Or simply rename `execFileSync` to `execAgd` and take out the `file` param :)